### DISCUSSION TITLE
How to build?

### DISCUSSION THREAD

**Discussion by benm5678** [artisan-roaster-scope/artisan#1760] [General]
Any tips on how to build this source code on a Mac?  I could not find in readme or get any of the scripts to work.  Thanks!

**Comment by roasterdave** [comment]
There is a document here on Github that details how you can duplicate the Continuous Integration (CI) builds that happen for each Artisan repository commit and pull request.  [HowToBuildArtisan.md](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/HowToBuildArtisan.md).  We recommend implementing this CI procedure.  The Artisan Team does not have the resources to support scripts for local builds.  If you want to build locally you must adapt the CI build scripts to your environment.  

Each Artisan release includes a notarized universal macOS binary that contains Intel/M1/M2/M3 binary code.  The procedure to generate the universal binary is not supported by scripts in the repository and thus is not presently available to be replicated.

The CI build procedure linked above should "work out of the box."  Let us know if that is not your experience.
